### PR TITLE
Specify allow round methods

### PR DESCRIPTION
**Pull request by ruudk**
This way, PHPStan can validate invalid input.

### CODE REVIEW

**Comment by ruudk** [comment]
@fabpot this one is also ready

**Comment by smnandre** [comment]
I think the return type is wrong too, as it is always a float

https://3v4l.org/aA0u3#v8.3.13

File: src/Extension/CoreExtension.php [hunk 1]
```diff
@@ -586,9 +586,9 @@ public static function replace($str, $from): string
...
      * Rounds a number.
      *
-     * @param int|float|string|null $value     The value to round
-     * @param int|float             $precision The rounding precision
-     * @param string                $method    The method to use for rounding
+     * @param int|float|string|null   $value     The value to round
+     * @param int|float               $precision The rounding precision
+     * @param 'common'|'ceil'|'floor' $method    The method to use for rounding
      *
      * @return int|float The rounded number
```
Comment (ruudk) [inline src/Extension/CoreExtension.php]:
```suggestion
     * @return float The rounded number
```

**Comment by ruudk** [comment]
@smnandre You're right. 

Also visible in the PHP docs http://php.net/round

Made the change.

**Review by stof — APPROVED** [review]

**Comment by ruudk** [comment]
@fabpot can this be merged?

**Comment by ruudk** [comment]
@fabpot Let's merge or close this one.

**Comment by fabpot** [comment]
Thank you @ruudk.

**Pull request merged**